common/v2/features/DeFiZap/components/ZapCard.tsx

```diff
@@ -53,14 +53,26 @@ const ZapCardContent = styled('div')`
   flex-direction: column;
 `;
 
-const ZapCardContentText = styled.div`
-  padding: ${SPACING.SM} 0px;
+const ZapCardContentText = styled.p`
```

Comment:
Is there a reason we don't use `Typography` for all of these texts? (in general throughout the flow)

Reply:
Afaict, we've not really stuck to using it. Didn't think it was that important of an addition :woman_shrugging: